packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionStatusInfo.tsx

```diff
@@ -52,14 +54,6 @@ export const TransactionStatusInfo = () => {
     )
   }, [transactions])
 
-  const shouldShow = useMemo(() => {
-    return (
-      numClaimableTransactions > 0 ||
-      numRetryablesToRedeem > 0 ||
-      numPendingTransactions > 0
-    )
-  }, [numClaimableTransactions, numRetryablesToRedeem, numPendingTransactions])
-
   const Content = useCallback(() => {
```

Comment:
now that i look at it, why isn't the `Content` a `React.memo` and is a `useCallback`?